### ISSUE TITLE
Anaconda Navigator does not start

### ISSUE THREAD

**Issue by ivan-paleo** [ContinuumIO/anaconda-issues#11985]
I have just installed the latest version of Anaconda.
I can launch the Anaconda Powershell Prompt, Anaconda Prompt and Jupyter Notebook, but the Anaconda Navigator and Sypder won't launch. No trace of them in the Task Manager either...

If I type `anaconda-navigator` in the Anaconda Prompt, I get the following message:
"PySide2/__init__.py: Unable to import shiboken2 from C:\Users\calandra.RGZM\Anaconda3\Scripts, C:\Users\calandra.RGZM\Anaconda3\python38.zip, C:\Users\calandra.RGZM\Anaconda3\DLLs, C:\Users\calandra.RGZM\Anaconda3\lib, C:\Users\calandra.RGZM\Anaconda3, C:\Users\calandra.RGZM\AppData\Roaming\Python\Python38\site-packages, C:\Users\calandra.RGZM\Anaconda3\lib\site-packages, C:\Users\calandra.RGZM\Anaconda3\lib\site-packages\win32, C:\Users\calandra.RGZM\Anaconda3\lib\site-packages\win32\lib, C:\Users\calandra.RGZM\Anaconda3\lib\site-packages\Pythonwin
Traceback (most recent call last):
  File "C:\Users\calandra.RGZM\Anaconda3\lib\site-packages\qtpy\__init__.py", line 204, in <module>
    from PySide import __version__ as PYSIDE_VERSION  # analysis:ignore
ModuleNotFoundError: No module named 'PySide'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\calandra.RGZM\Anaconda3\Scripts\anaconda-navigator-script.py", line 6, in <module>
    from anaconda_navigator.app.main import main
  File "C:\Users\calandra.RGZM\Anaconda3\lib\site-packages\anaconda_navigator\app\main.py", line 22, in <module>
    from anaconda_navigator.utils.conda import is_conda_available
  File "C:\Users\calandra.RGZM\Anaconda3\lib\site-packages\anaconda_navigator\utils\__init__.py", line 15, in <module>
    from qtpy.QtGui import QIcon
  File "C:\Users\calandra.RGZM\Anaconda3\lib\site-packages\qtpy\__init__.py", line 210, in <module>
    raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found"

I'm using Windows 10 Pro, version 2004, OS build 19041.388, Windows Feature Experience Pack 120.2202.130.0.
Any help appreciated! Thanks.

**Comment by ivan-paleo** [comment]
Apparently, the issue is only with the 64-bit version of the installer.
I installed the 32-bit version exactly the same way and the Anaconda Navigator launches just fine.

**Comment by rediffusion** [comment]
@ivan-paleo 

 Yes, you're right but I got some errors, take a look please:

 When I run **spyder**:

![image](https://user-images.githubusercontent.com/30071027/91012780-54bf7780-e5ef-11ea-90bf-8dad77288bb6.png)

... **jedi** and **parso** already installed if I go to Anaconda Navigator > Environments > installed:

![image](https://user-images.githubusercontent.com/30071027/91013149-ee872480-e5ef-11ea-802f-547b5b669a2d.png)
![image](https://user-images.githubusercontent.com/30071027/91013380-49208080-e5f0-11ea-80ed-0c9f34434434.png)

 When I run **Jupyter Notebook**:

<details>

```
[W 10:04:07.238 NotebookApp] Terminals not available (error was No module named 'winpty.cywinpty')
[I 10:04:07.696 NotebookApp] JupyterLab extension loaded from C:\ProgramData\Anaconda3\lib\site-packages\jupyterlab
[I 10:04:07.696 NotebookApp] JupyterLab application directory is C:\ProgramData\Anaconda3\share\jupyter\lab
[I 10:04:07.699 NotebookApp] Serving notebooks from local directory: C:\Users\NIKA
[I 10:04:07.699 NotebookApp] Jupyter Notebook 6.1.3 is running at:
[I 10:04:07.699 NotebookApp] http://localhost:8888/?token=2b6e3087bdd6340714405e9ddfa0567fe6bbdd6b39f37677
[I 10:04:07.700 NotebookApp] or http://127.0.0.1:8888/?token=2b6e3087bdd6340714405e9ddfa0567fe6bbdd6b39f37677
[I 10:04:07.700 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 10:04:07.780 NotebookApp] 

To access the notebook, open this file in a browser:
file:///C:/Users/NIKA/AppData/Roaming/jupyter/runtime/nbserver-5872-open.html
Or copy and paste one of these URLs:
http://localhost:8888/?token=2b6e3087bdd6340714405e9ddfa0567fe6bbdd6b39f37677
or http://127.0.0.1:8888/?token=2b6e3087bdd6340714405e9ddfa0567fe6bbdd6b39f37677
```

</details>

**Comment by ivan-paleo** [comment]
I gave up on Anaconda: did not work, nobody to help, and I'm using Python through RStudio anyway (together with the R package reticulate). So I just installed plain Python and everything is running fine.

**Comment by rediffusion** [comment]
@ivan-paleo 

1 month ago I installed «Windows 10 2004» 2 days ago I **clean** my PC (cooler, CPU Fan, video card, etc.).
Then I use «IObit Driver Booster» to **update drivers**. After this, I can't launch the "Anaconda Navigator".
Surf the internet and find that need to install [Microsoft Visual C++ 2010 Redistributable Package](https://chocolatey.org/packages/vcredist2010#install). I tried but with no success!

Thanks, bro for your response anyway.  😊😇👍

**Comment by ivan-paleo** [comment]
I have found tons of outdated solutions. I guess the installer(s) really have a problem...
Good luck!

**Comment by meRscliche** [comment]
(1) I encounter exactly the same problem as you did on my NB(Win10 & Anaconda 2020.07) today.
After googling, somehow I managed to solve it, but I got to launch Jupyter and Spyder in Anaconda Navigator from now on. Well, it's better than never launching Jupyter and Spyder.

(2) Launch the Anaconda Prompt as a system administrator, then enter the following line in order:

conda update conda
conda update --all
conda remove QtPy PyQt5
pip install QtPy PyQt5
ERROR: spyder 4.1.4 requires pyqtwebengine<5.13; python_version >= "3", which is not installed.
ERROR: spyder 4.1.4 has requirement pyqt5<5.13; python_version >= "3", but you'll have pyqt5 5.15.1 which is incompatible.
The errors above are just warnings and won't stop the process.

conda update anaconda-navigator
anaconda-navigator --reset
conda update anaconda-client
conda update -f anaconda-client

**Comment by meRscliche** [comment]
> @ivan-paleo
> 
> 1 month ago I installed «Windows 10 2004» 2 days ago I **clean** my PC (cooler, CPU Fan, video card, etc.).
> Then I use «IObit Driver Booster» to **update drivers**. After this, I can't launch the "Anaconda Navigator".
> Surf the internet and find that need to install [Microsoft Visual C++ 2010 Redistributable Package](https://chocolatey.org/packages/vcredist2010#install). I tried but with no success!
> 
> Thanks, bro for your response anyway. 😊😇👍



> I gave up on Anaconda: did not work, nobody to help, and I'm using Python through RStudio anyway (together with the R package reticulate). So I just installed plain Python and everything is running fine.

Thanks for the info. Seems that I shall try Rstudio instead.

**Comment by muhrizqiardi** [comment]
TL;DR I fixed it by installing the 32-bit version.

I don't know about the others, but I fixed it by uninstalling the current one and install the 32bit version. Is this intentional? Is it possible that the 64-bit installer has a bug or something? Or is there something from my device that made this happen? Genuinely curious.

**Comment by ivan-paleo** [comment]
> 
> 
> TL;DR I fixed it by installing the 32-bit version.
> 
> I don't know about the others, but I fixed it by uninstalling the current one and install the 32bit version. Is this intentional? Is it possible that the 64-bit installer has a bug or something? Or is there something from my device that made this happen? Genuinely curious.

That's exactly what I did one year ago (see 2nd post)... It seems the issue hasn't been resolved in > 1 year... It says a lot about the software...
